pettingzoo/classic/tictactoe/tictactoe.py

```diff
@@ -221,34 +247,74 @@ def reset(self, seed=None, options=None):
         self._agent_selector.reset()
         self.agent_selection = self._agent_selector.reset()
 
+    def close(self):
+        pass
+
     def render(self):
         if self.render_mode is None:
             gymnasium.logger.warn(
                 "You are calling render method without specifying any render mode."
             )
             return
 
+        screen_height = self.screen_height
+        screen_width = self.screen_height
+
+        # TODO: refactor this and check if pygame.font init needs to be done
+        # Ideally this should look like all the other environments
+        if self.render_mode == "human":
+            if self.screen is None:
+                pygame.init()
+                self.screen = pygame.display.set_mode((screen_width, screen_height))
+                pygame.display.set_caption("Tic-Tac-Toe")
+        elif self.screen is None:
+            pygame.font.init()
+            self.screen = pygame.Surface((screen_width, screen_height))
+
+        # Setup dimensions for 'x' and 'o' marks
+        tile_size = int(screen_height / 4)
+
+        # Load and blit the board image for the game
+        board_img = get_image(os.path.join("img", "board.png"))
+        board_img = pygame.transform.scale(
+            board_img, (int(screen_width), int(screen_height))
+        )
+
+        self.screen.blit(board_img, (0, 0))
+
+        # Load and blit actions for the game
         def getSymbol(input):
             if input == 0:
-                return "-"
+                return None
             elif input == 1:
-                return "X"
+                return "cross"
             else:
-                return "O"
+                return "circle"
 
-        board = list(map(getSymbol, self.board.squares))
+        board_state = list(map(getSymbol, self.board.squares))
```

Comment:
Agreed with renaming to `board_state`, simpler to understand without looking too deep into the code